### ISSUE TITLE
[Adaptive Cards - Input]: Talkback is announcing incorrect role as 'Button' for the 'Reply' link under 'Input' screen.

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards-Mobile#104]
### Target Platforms

Android

### SDK Version

2023.12.12.1 (85454887)

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
OS: Android 13
Device: S20
Screen Reader: Talkback
AdaptiveCard Android App version: 2023.12.12.1 (85454887)

**Repro Steps:**
1. Launch the Adaptive cards app on Android and turn on the Talkback.
2. Navigate to 'Import' control and select it.
3. Navigate to 'Sample cards' and select it.
4. Navigate to available dropdown under 'Sample cards' and select it.
5. Navigate all the controls under dropdown and select 'Stock Update'.
6. Navigate to 'Visual' tab control and select it.
7. Navigate to 'Input' option controls under 'Visual'.
8. Navigate the elements present under 'Inputs' and observe the issue.

**Actual Result:**
While navigating to the 'Reply' option, Talkback is announcing the incorrect role as 'Button' as on activating it, a new web page is getting open on the browser.

**Expected Result:**
Talkback should announce the correct role for the 'Reply' option and when the focus lands on it then Talkback should announce the information as 'Link' it should announce as **'Reply link double tap to activate it.'**

**User Impact:**
It will impact the screen reader user as they will get impacted if the incorrect role is defined for any control on the screen. If the role is defined as button then they will assume as after activating it information will display on the same page.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/name-role-value

Have feedback to share on Bugs ? please tag bug as “A11yRCA” and add your feedback in the comment section.

### Screenshots

https://github.com/microsoft/AdaptiveCards-Mobile/assets/95913748/c4553888-0391-404d-998f-c9af85aae0eb

![Talkback is announcing incorrect role as 'Button' for the 'Reply' link under 'Input' screen](https://github.com/microsoft/AdaptiveCards-Mobile/assets/95913748/cba00816-8ce9-40c7-bc9a-6293a310fb47)

### Card JSON

```JSON
NA
```

**Comment by vagpt** [comment]
This issue is partially fixed as of now screen reader is announce twice role as link and button and for the same we have logged a new bug #201 

![image](https://github.com/user-attachments/assets/892e76ea-3c98-4e58-bcbc-f8cc6b617aeb)
